calico_node/filesystem/etc/rc.local

```diff
@@ -27,12 +27,22 @@ else
 
     # Run confd twice.  Our confd TOML files are also generated from confd, so
     # running twice ensures our starting configuration is correct.
-    confd -confdir=/etc/calico/confd -onetime -node=${ETCD_SCHEME:=http}://${ETCD_AUTHORITY} \
+    case "$ETCD_ENDPOINTS" in
```

Comment:
I don't get this craziness. Why are you pulling out the ETCD_SCHEME? And why the two different cases?